src/mito2/src/worker/handle_alter.rs

```diff
@@ -167,6 +169,92 @@ impl<S> RegionWorkerLoop<S> {
                         current_options.ttl = Some(new_ttl);
                     }
                 }
+                ChangeOption::TwscMaxActiveWindowRuns(runs) => {
+                    let Twcs(options) = current_options.compaction;
+                    let runs = runs.unwrap_or(TwcsOptions::default().max_active_window_runs);
+                    info!(
+                        "Update region compaction.twcs.max_active_window_runs: {}, previous: {} new: {}",
+                        region.region_id, options.max_active_window_runs, runs
+                    );
+                    let mut new_option = options.clone();
+                    new_option.max_active_window_runs = runs;
+                    current_options.compaction = Twcs(new_option)
```

Comment:
Can we avoid the clone here?

```suggestion
                    let Twcs(options) = &mut current_options.compaction;
                    let runs = runs.unwrap_or(TwcsOptions::default().max_active_window_runs);
                    info!(
                        "Update region compaction.twcs.max_active_window_runs: {}, previous: {} new: {}",
                        region.region_id, options.max_active_window_runs, runs
                    );
                    options.max_active_window_runs = runs;
```

Reply:
good call fixed it